### PR TITLE
Feature/hg 8 admin page

### PR DESCRIPTION
**Pull request by NguyenChiTrung1310**
**[HG-8] [ADMIN][FE] Home Page - Structure Layout**
- Create sidebar menu.
- Basict content.

[HG-8]: https://paper-coding22.atlassian.net/browse/HG-8

### CODE REVIEW

File: src/pages/admin/AdminPage.scss [hunk 1]
```diff
@@ -0,0 +1,176 @@
+@import '../../styles/_mixin';
```
Comment (papercoding) [inline src/pages/admin/AdminPage.scss:1]: Did you use any mixin function in this css file, if not remove this import

File: src/pages/login/LoginPage.tsx [hunk 2]
```diff
@@ -116,4 +116,4 @@ const LoginPage = () => {
   );
 };
 
-export default LoginPage;
+export { LoginPage };
```
Comment (papercoding) [inline src/pages/login/LoginPage.tsx]: WTF, it should be `export default`, why you changed it ?

File: src/pages/admin/AdminPage.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          </SubMenu>
+          <Menu.Item key='9' icon={<FileOutlined />} />
+        </Menu>
+      </Sider>
+      <Layout className='site-layout'>
+        <Header className='site-layout-background'>
+          <div className='site-layout-left'>
+            <Search
+              placeholder='Search for...'
+              onSearch={(value) => console.log(value)}
```
Comment (papercoding) [inline src/pages/admin/AdminPage.tsx]: Remove console log

File: src/pages/admin/AdminPage.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+          </SubMenu>
+          <SubMenu key='sub2' icon={<TeamOutlined />} title='Team'>
+            <Menu.Item key='6'>Team 1</Menu.Item>
+            <Menu.Item key='8'>Team 2</Menu.Item>
+          </SubMenu>
+          <Menu.Item key='9' icon={<FileOutlined />} />
+        </Menu>
+      </Sider>
+      <Layout className='site-layout'>
+        <Header className='site-layout-background'>
```
Comment (papercoding) [inline src/pages/admin/AdminPage.tsx]: Why component `Header` has className `site-layout-background` ??? It should be `header-layout` or something more relative

File: src/pages/admin/AdminPage.scss (hunk 1 above)
Comment (NguyenChiTrung1310) [reply src/pages/admin/AdminPage.scss:1]: @papercoding Please review again ! 
Screen Shot 2020-09-27 at 19 50 30

File: src/pages/login/LoginPage.tsx (hunk 2 above)
Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx:119]: @papercoding check inb messenger!

Comment (NguyenChiTrung1310) [reply src/pages/login/LoginPage.tsx]: @papercoding Fixed!

File: src/pages/admin/AdminPage.tsx (hunk 3 above)
Comment (NguyenChiTrung1310) [reply src/pages/admin/AdminPage.tsx]: @papercoding Fixed and removed!

File: src/pages/admin/AdminPage.tsx (hunk 4 above)
Comment (NguyenChiTrung1310) [reply src/pages/admin/AdminPage.tsx]: @papercoding Fixed and changed to "header-layout"

**Review by papercoding — APPROVED** [review]

**Pull request merged**